src/test/resources/outputs/trino/basics/datetime.sql

```diff
@@ -0,0 +1,48 @@
+-- DATE
```

Comment:
Fortunately, Trino seems to have the same parsing and evaluation behavior as PartiQL for datetime types.